src/activities/pdfUtils.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Converts a color hex code into RGBA values.
+ * @param hex The color hex code.
+ * @returns RGBA values between 0 and 1.
+ */
+export function hexToRgba(hex = "000000FF") {
+    hex = hex.replace("#", "");
```

Comment:
Should this be stricter? Only a single leading `#` is valid, whereas this would strip them from anywhere.

Reply:
Done

Reply:
I've added tests too.